.github/workflows/ci.yaml

```diff
@@ -48,15 +51,33 @@ jobs:
         name: codecov-umbrella # optional
         fail_ci_if_error: true # optional (default = false)
 
-    - name: Docker build
-      if: github.event_name == 'pull_request'
+    - name: Docker build # on PRs. on pushes to forks
```

Comment:
Can we clarify this comment?

Reply:
I think you mean somthing like "Triggers on submitted pull requests or merged pull requests"

Reply:
(Or just remove the comment since the if logic below is pretty clear)

Reply:
That means that the step will run on PRs regardless of the repo it is at. And will also run on pushes when its not in `keikoproj/upgrade-manager`, so forks. Maybe `runs on PRs. Only on pushes when not in keikoproj/upgrade-manager`?

Reply:
Sure, lets remove it